packages/Panel/stories/examples/FromBottom.js

```diff
@@ -1,11 +1,13 @@
 import React from "react";
+import tokens from "@paprika/tokens";
+import Button from "@paprika/button";
 import Panel from "../../src";
 import { TextLine } from "../helpers";
 
 export default function FromBottom() {
   const [isFooterOpen, setIsFooterOpen] = React.useState(true);
   const [isNavOpen, setIsNavOpen] = React.useState(true);
-  const navWidth = { expanded: 300, collapsed: 30 };
+  const navWidth = { expanded: 295, collapsed: 30 };
```

Comment:
```suggestion
  const navWidth = { expanded: 295, collapsed: 32 };
```

Reply:
This is off by just a little bit when collapsed.

Reply:
https://user-images.githubusercontent.com/45611276/148741474-da217c30-4053-43bc-8acf-ceade3e4ff1e.mov

It looks more off to me.....🤔